PC/winreg.c

```diff
@@ -580,6 +580,7 @@ Py2Reg(PyObject *value, DWORD typ, BYTE **retDataBuf, DWORD *retDataSize)
             else {
                 DWORD d = PyLong_AsUnsignedLong(value);
                 if ((d == (DWORD)(-1)) && PyErr_Occurred()) {
+                    free(retDataBuf);
```

Comment:
```suggestion
                    PyMem_Free(retDataBuf);
```

Reply:
Ditto for the other `free`.

Reply:
Fixed in commit 78417143294ff843d58dd61d1fb7e190227566b5

Reply:
Thanks :)